server/controllers/airlines.js

```diff
@@ -0,0 +1,19 @@
+const Airline = require('../models').Airline;
+
+module.exports = {
+  create(req, res) {
+    return Airline
+      .create({
+        name: req.body.name,
```

Comment:
We don't validate the name. We shouldn't be allowed to create an airline without a name.
We can fix this in two ways:

1. We check in the `create()` function if the name exists and throw an error if it doesn't
2. We update the model with the code below and sequelize will throw a `400 - Bad request` out of the box (This is the nicest way)
```
    name: {
      type: DataTypes.STRING,
      allowNull: false
    }
```